Google.PowerShell.IntegrationTests/Container/Container.GkeCluster.Tests.ps1

```diff
@@ -162,10 +184,151 @@ Describe "New-GkeNodeConfig" {
     It "should raise an error for wrong DiskSize" {
         { New-GkeNodeConfig -DiskSizeGb 3 } | Should Throw "less than the minimum allowed range of 10"
     }
+}
+
+Describe "Add-GkeCluster" {
+    $r = Get-Random
+
+    # Create the cluster in parallel to reduce wait time.
```

Comment:
How much time does this take? I find this fairly confusing (and confusing to maintain) to have individual test set up here